2022-02-28 15:49:42,709 INFO    MainThread:625774 [wandb_setup.py:_flush():71] setting env: {}
2022-02-28 15:49:42,709 INFO    MainThread:625774 [wandb_setup.py:_flush():71] setting login settings: {}
2022-02-28 15:49:42,709 INFO    MainThread:625774 [wandb_setup.py:_flush():71] setting login settings: {}
2022-02-28 15:49:42,709 INFO    MainThread:625774 [wandb_init.py:_log_setup():371] Logging user logs to /home/junshick/Workspace/pose_estimation/custom_pose/wandb/run-20220228_154942-2gtrlik7/logs/debug.log
2022-02-28 15:49:42,709 INFO    MainThread:625774 [wandb_init.py:_log_setup():372] Logging internal logs to /home/junshick/Workspace/pose_estimation/custom_pose/wandb/run-20220228_154942-2gtrlik7/logs/debug-internal.log
2022-02-28 15:49:42,709 INFO    MainThread:625774 [wandb_init.py:init():404] calling init triggers
2022-02-28 15:49:42,709 INFO    MainThread:625774 [wandb_init.py:init():411] wandb.init called with sweep_config: {}
config: {}
2022-02-28 15:49:42,709 INFO    MainThread:625774 [wandb_init.py:init():460] starting backend
2022-02-28 15:49:42,709 INFO    MainThread:625774 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-02-28 15:49:42,716 INFO    MainThread:625774 [backend.py:ensure_launched():216] starting backend process...
2022-02-28 15:49:42,719 INFO    MainThread:625774 [backend.py:ensure_launched():222] started backend process with pid: 625783
2022-02-28 15:49:42,720 INFO    MainThread:625774 [wandb_init.py:init():469] backend started and connected
2022-02-28 15:49:42,721 INFO    MainThread:625774 [wandb_init.py:init():533] updated telemetry
2022-02-28 15:49:42,723 INFO    MainThread:625774 [wandb_init.py:init():563] communicating current version
2022-02-28 15:49:43,189 INFO    MainThread:625774 [wandb_init.py:init():568] got version response upgrade_message: "wandb version 0.12.10 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-02-28 15:49:43,189 INFO    MainThread:625774 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-02-28 15:49:43,541 INFO    MainThread:625774 [wandb_init.py:init():606] starting run threads in backend
2022-02-28 15:49:46,182 INFO    MainThread:625774 [wandb_run.py:_console_start():1810] atexit reg
2022-02-28 15:49:46,183 INFO    MainThread:625774 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.REDIRECT
2022-02-28 15:49:46,183 INFO    MainThread:625774 [wandb_run.py:_redirect():1689] Redirecting console.
2022-02-28 15:49:46,184 INFO    MainThread:625774 [wandb_run.py:_redirect():1745] Redirects installed.
2022-02-28 15:49:46,184 INFO    MainThread:625774 [wandb_init.py:init():633] run started, returning control to user process
2022-02-28 15:49:51,496 INFO    MainThread:625774 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 255
2022-02-28 15:49:51,497 INFO    MainThread:625774 [wandb_run.py:_restore():1752] restore
2022-02-28 15:49:53,703 INFO    MainThread:625774 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 681
  total_bytes: 681
}

2022-02-28 15:49:54,033 INFO    MainThread:625774 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 681
  total_bytes: 681
}

2022-02-28 15:49:54,554 INFO    MainThread:625774 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 681
  total_bytes: 15956
}

2022-02-28 15:49:54,657 INFO    MainThread:625774 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 681
  total_bytes: 15956
}

2022-02-28 15:49:54,759 INFO    MainThread:625774 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 681
  total_bytes: 15956
}

2022-02-28 15:49:54,861 INFO    MainThread:625774 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 681
  total_bytes: 15956
}

2022-02-28 15:49:54,963 INFO    MainThread:625774 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15956
  total_bytes: 15956
}

2022-02-28 15:49:55,066 INFO    MainThread:625774 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15956
  total_bytes: 15956
}

2022-02-28 15:49:55,169 INFO    MainThread:625774 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15956
  total_bytes: 15956
}

2022-02-28 15:49:55,271 INFO    MainThread:625774 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15956
  total_bytes: 15956
}

2022-02-28 15:49:55,374 INFO    MainThread:625774 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15956
  total_bytes: 15956
}

2022-02-28 15:49:55,751 INFO    MainThread:625774 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15956
  total_bytes: 15956
}

2022-02-28 15:49:56,356 INFO    MainThread:625774 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15956
  total_bytes: 15956
}
local_info {
}

2022-02-28 15:49:57,422 INFO    MainThread:625774 [wandb_run.py:_append_history():2130] rendering history
2022-02-28 15:49:57,423 INFO    MainThread:625774 [wandb_run.py:_append_summary():2085] rendering summary
2022-02-28 15:49:57,423 INFO    MainThread:625774 [wandb_run.py:_append_files():2180] logging synced files
